docs/src/main/asciidoc/getting-started-testing.adoc

```diff
@@ -828,13 +828,13 @@ Note that there is no dependency on Mockito, you can use any mocking library you
 objects to provide the behaviour you require.
 
 NOTE: Using `@Inject` will get you a CDI proxy to the mock instance you install, which is not suitable for passing to methods such as `Mockito.verify`
-which want the mock instance itself. So if you need to call methods such as `verify` you need to hang on to the mock instance in your test, or use `@InjectMock`
+which want the mock instance itself. So if you need to call methods such as `verify` you need to hang on to the mock instance in your test, or use `@io.quarkus.test.InjectMock`
 as shown below.
 
 ==== Further simplification with `@InjectMock`
 
 Building on the features provided by `QuarkusMock`, Quarkus also allows users to effortlessly take advantage of link:https://site.mockito.org/[Mockito] for mocking the beans supported by `QuarkusMock`.
-This functionality is available via the `@io.quarkus.test.junit.mockito.InjectMock` annotation which is available in the `quarkus-junit5-mockito` dependency.
+This functionality is available via the `@io.quarkus.test.InjectMock` annotation.
```

Comment:
Do people also need to have the `quarkus-junit5-mockito` dependency in their project, if they want mockito to be doing the mocking?

Reply:
Oh, and for the linter comment, IANATW, but I'd suggest 'with', rather than 'via' or 'through'. 

"This functionality is available with the `@io.quarkus.test.InjectMock` annotation."

I'm not totally happy with 'with', though, so happy to be over-ruled.

Reply:
Yes, the `quarkus-junit5-mockito` dependency is still needed. So what about:

"This functionality is available with the `@io.quarkus.test.InjectMock` annotation if the `quarkus-junit5-mockito` dependency is present."

Reply:
If they have `quarkus-junit5-component` and not `quarkus-junit-mockito`, is the behaviour that `@InjectMock` would work in a `@QuarkusComponentTest` (since there's actually a `@ConfigureMock` in disguise :) ), but not work in other tests? 

That could be confusing, but I have no idea what could be done to make it less confusing.